src/Tests/Snippets.cs

```diff
@@ -37,7 +37,11 @@ public static async Task SyncPr(ICredentials octokitCredentials)
             branch: "master"));
 
         // Run the sync
-        await repoSync.Sync(syncOutput: SyncOutput.MergePullRequest);
```

Comment:
can this be part of a new snippet

Reply:
should prTitle, branchName, & commitMessage for the Sync method in the RepoSync class be optional params with default values?